udfs/tests/run.sh

```diff
@@ -14,6 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
```

Comment:
this seems like an unintentional change.

Reply:
Reverted.